[2024-06-02T15:00:04.907+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: anomaly_detection_and_alerting.send_alert scheduled__2024-06-02T14:00:00+00:00 [queued]>
[2024-06-02T15:00:04.921+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: anomaly_detection_and_alerting.send_alert scheduled__2024-06-02T14:00:00+00:00 [queued]>
[2024-06-02T15:00:04.923+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-06-02T15:00:04.940+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): send_alert> on 2024-06-02 14:00:00+00:00
[2024-06-02T15:00:04.946+0000] {standard_task_runner.py:60} INFO - Started process 6549 to run task
[2024-06-02T15:00:04.950+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'anomaly_detection_and_alerting', 'send_alert', 'scheduled__2024-06-02T14:00:00+00:00', '--job-id', '688', '--raw', '--subdir', 'DAGS_FOLDER/alert.py', '--cfg-path', '/tmp/tmpjwsmws6f']
[2024-06-02T15:00:04.952+0000] {standard_task_runner.py:88} INFO - Job 688: Subtask send_alert
[2024-06-02T15:00:05.025+0000] {task_command.py:423} INFO - Running <TaskInstance: anomaly_detection_and_alerting.send_alert scheduled__2024-06-02T14:00:00+00:00 [running]> on host 6f8b2a20b5b0
[2024-06-02T15:00:05.230+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='anomaly_detection_and_alerting' AIRFLOW_CTX_TASK_ID='send_alert' AIRFLOW_CTX_EXECUTION_DATE='2024-06-02T14:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-02T14:00:00+00:00'
[2024-06-02T15:00:05.263+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-06-02T15:00:05.283+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=anomaly_detection_and_alerting, task_id=send_alert, execution_date=20240602T140000, start_date=20240602T150004, end_date=20240602T150005
[2024-06-02T15:00:05.325+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-06-02T15:00:05.356+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
